src/consts/timebarConsts.js

```diff
@@ -63,3 +65,8 @@ export const timebarFormat = {
     }
   }
 };
+
+/**
+ * Default column header renderer
+ */
+export const headerRenderer = () => <div />;
```

Comment:
not a good place to store components. separate file: `ColumnHeaderRenderer.js`. It will have some logic (e.g. extract from `column.headerLabel`), so let's transform it into a class.